war/src/main/js/config-tabbar.js

```diff
@@ -94,3 +99,22 @@ function addFinderToggle(configTableMetadata) {
 function fireBottomStickerAdjustEvent() {
     Event.fire(window, 'jenkins:bottom-sticker-adjust'); // jshint ignore:line
 }
+// YUI Drag widget does not like to work on elements with a relative position.
+// This tells the element to switch to static position at the start of the drag, so it can work.
+function fixDragEvent(handle){
+    var isReady = false;
+    var $handle = $(handle);
+    var $chunk = $handle.closest('.repeated-chunk');
+    $handle.add('#ygddfdiv')
+    	.mousedown(function(){
+    	    isReady = true; 
+    	})
+    	.mousemove(function(){
+    	    if(isReady && !$chunk.hasClass('dragging')){
```

Comment:
This is related to the above comment as well, yeah it is hacky, but I need the parent box to have a relative position, but the YUI drag needs it to be static, thus I need to hack in the switch.